### ISSUE TITLE
`corectl display-http-requests`: invalid log on POST method

### ISSUE THREAD

**Issue by nil0x42** [nil0x42/phpsploit#65]
Steps to reproduce:
```sh
# use POST method:
set REQ_DEFAULT_METHOD "POST"
# execute a simple remote request:
run 'whoami'
# see last request log
corectl 'display-http-requests'
```
Last command should show the last request output, instead, the header appears as 1st request, while body appears as the 2nd one:
```
##############################################################################
### REQUEST 1
##############################################################################
POST /admin/index.php HTTP/1.1
Accept-Encoding: identity
Content-Type: application/x-www-form-urlencoded
Content-Length: 666
Passkey: <HEADER_PAYLOAD_STAGER>
Host: <HOSTNAME>
User-Agent: <AGENT>
Connection: close


##############################################################################
### REQUEST 2
##############################################################################
Passkey=<BASE64_PAYLOAD>
```

**Comment by nil0x42** [comment]
a simple (and dirty) fix could be to concatenate a request that doesn't start with "^(GET|POST) /"  with the previous one on `./src/ui/interface.py:266`

**Comment by nil0x42** [comment]
fixed by 4010bac
